Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -1143,6 +1271,19 @@ class CombinatorialKalmanFilter {
     // copy source link accessor, calibrator and measurement selector
     combKalmanActor.m_sourcelinkAccessor = tfOptions.sourcelinkAccessor;
     combKalmanActor.m_extensions = tfOptions.extensions;
+    combKalmanActor.trackStateCandidateCreator =
+        tfOptions.trackStateCandidateCreator;
+    DefaultTrackStateCreator defaultTrackStateCreator;
```

Comment:
can this be moved inside the `if` block?

Reply:
why not